src/client.ts

```diff
@@ -108,6 +107,8 @@ export class SkynetClient {
           config.onUploadProgress(progress, event);
         },
       timeout: config.timeout,
+      transformRequest: config.transformRequest,
+      transformResponse: config.transformResponse,
```

Comment:
this is not something devs will be able to override right? we don't want to expose that to devs

Reply:
Since we agree, I'll make `executeRequest` protected as we are breaking compatibility anyway.

The main use of that for devs would have been to get file contents (not sure if it was actually used that way). We have that coming soon in Dan's PR.